model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -104,7 +104,7 @@ class HospitalAdmissions(RandomVariable):
 
     def __init__(
         self,
-        inf_hosp_int: RandomVariable,
+        inform_hosp: RandomVariable,
```

Comment:
`infection_to_hospitalization_interval`

Reply:
Is that the same as the delay? In the equation we have:

$$
H(t) = \omega(t) p_\mathrm{hosp}(t) \sum_{\\tau = 0}^{T_d} d(\tau) I(t-\tau
$$

where:

- $\omega(t)$: `weekday_effect_dist`
- $p_\mathrm{hosp}(t)$: `p_hosp_dist`
- $I(t)$: IHR (`infect_hosp_rate_dist`) * `random_variables.get(self.infections_varname)`
- $d(\tau)$: `inf_hosp_int` (delay)

Here is the code:

https://github.com/CDCgov/multisignal-epi-inference/blob/517fce1ab66c8d589577f027f6dcaf4eebd3733f/model/src/pyrenew/latent/hospitaladmissions.py#L194-L208

Reply:
Yes; it's the delay distribution from infection to (reported) admission, represented as a discrete PMF

Reply:
OK, renamed in [ddac288](https://github.com/CDCgov/multisignal-epi-inference/pull/39/commits/ddac28844aba2e66c6d5ce1c1b7369965f569c77)